substratevm/src/com.oracle.svm.configure/src/com/oracle/svm/configure/trace/AccessAdvisor.java

```diff
@@ -67,16 +67,18 @@ public static RuleNode copyBuiltinCallerFilterTree() {
         return internalCallerFilter.copy();
     }
 
+    public static RuleNode copyBuiltinAccessFilterTree() {
+        RuleNode root = RuleNode.createRoot();
+        root.addOrGetChildren("**", RuleNode.Inclusion.Include);
```

Comment:
A small question - could this also work like `copyBuiltinCallerFilterTree`?  It would copy `accessWithoutCallerFilter`, and then we could remove
```
        if (callerClass.get() == null && !accessWithoutCallerFilter.treeIncludes(queriedClass.get())) {
            return true;
        }
```
from the `shouldIgnore` call?

Reply:
I'm not sure if I understand correctly. It would probably be fine to merge the two filters, but `accessWithoutCallerFilter` is specifically for cases in which we don't know the caller class and it's very likely an internal access (JVMCI, Libgraal, JVM startup) where we assume it shouldn't end up in the user configuration, so I'd prefer to keep it separate.

Reply:
Ahh, I overlooked the purpose initially, my bad. I agree, it's best to keep it separate.